jsonschema/_validators.py

```diff
@@ -391,6 +391,15 @@ def type(validator, types, instance, schema):
         yield ValidationError(_utils.types_msg(instance, types))
 
 
+def type_draft6(validator, types, instance, schema):
```

Comment:
Here too I think draft 6 taught us we need to widen the type interface. I put some notes in #364, maybe have a look there if you don't mind?